src/LanguageLine.php

```diff
@@ -27,20 +27,28 @@ public static function boot()
         });
     }
 
+    /**
+     * Get translations for group
+     *
+     * @param string $locale
+     * @param string $group
+     *
+     * @return array
+     * @see https://adamwathan.me/2016/07/14/customizing-keys-when-mapping-collections/
+     */
     public static function getTranslationsForGroup(string $locale, string $group): array
     {
         return Cache::rememberForever(static::getCacheKey($group, $locale), function () use ($group, $locale) {
-            return static::query()
+            $lines = static::query()
```

Comment:
Almost there. Could we just return this collection chain?

Is that final `$lines ?? []` really necessary?

Try out those modification and see if the tests still pass.